backend/src/storage.ts

```diff
@@ -1,28 +1,127 @@
 import * as fs from 'fs';
 import * as fsp from 'fs/promises';
 import * as path from 'path';
+import { Database } from 'sqlite3';
 
 export const STORAGE_FOLDER = process.env['TMT_STORAGE_FOLDER'] || 'storage';
+export const DATABASE_PATH = path.join(STORAGE_FOLDER, 'database.sqlite');
+export const GAME_SERVERS_TABLE = 'gameServers';
+export const PLAYERS_TABLE = 'players';
+export const MATCHES_TABLE = 'matches';
+export const PLAYER_MATCH_STATS_TABLE = 'playerMatchStats';
```

Comment:
Wouldn't it be better to create the tables from the related files instead of storage.ts ? This is pretty specific

Like there should be a method to create a table and you call that from every file that needs to handle a DB

Reply:
I'll need to make a full new class to pass the arguments, but it makes sense, yes